orchestration/dagster_orchestration/hca_manage/common.py

```diff
@@ -25,16 +37,8 @@ def has_problems(self) -> bool:
         return self.duplicates > 0 or self.null_file_refs > 0 or self.dangling_project_refs > 0
 
 
-data_repo_host = {
-    "dev": "https://jade.datarepo-dev.broadinstitute.org/",
-    "prod": "https://jade-terra.datarepo-prod.broadinstitute.org/",
-    "real_prod": "https://data.terra.bio/"
-}
-
-data_repo_profile_ids = {
-    "dev": "390e7a85-d47f-4531-b612-165fc977d3bd",
-    "prod": "db61c343-6dfe-4d14-84e9-60ddf97ea73f"
-}
+def setup_cli_logging_format():
+    logging.basicConfig(level=logging.INFO, format='%(message)s')
```

Comment:
I am centralizing our logging format for CLI tools here, and ditching the timestamps + loglevel as I find them a bit noisy.